CHANGELOG.md

```diff
@@ -10,6 +10,12 @@ Note that Sockeye has checks in place to not translate with an old model that wa
 
 Each version section may have have subsections for: _Added_, _Changed_, _Removed_, _Deprecated_, and _Fixed_.
 
+## [1.18.14]
+
+### Added
+- Introduced Sockeye Autopilot for single-command end-to-end system building.
```

Comment:
Can we mention that this module was added to `contrib`, meaning that it is not tested with every PR?

Reply:
Sure, we can be clear about that in the documentation.